docs/user-guides/notifications/catalog/templates.md

```diff
@@ -0,0 +1,101 @@
+# Triggers Catalog
+
+This document provides a catalog of recommended [templates] for common use cases
+in Config Sync. This catalog uses email as an example, but these can be modified
+to work for other services as well. See the [services] reference doc for the
+template configuration API of each service.
+
+The set of [built-in functions](./functions.md) can be invoked from templates.
+
+## reconciler-created
+
+```yaml
+  template.reconciler-created: |
+    message: |
+      {{.sync.kind}} {{.sync.metadata.namespace}}/{{.sync.metadata.name}} has been created.
+    email:
+      subject: |
+        A new {{.sync.kind}} has been created.
```

Comment:
The subject can be the same as message. Ditto for other templates.

Reply:
done

Reply:
https://github.com/argoproj/argo-cd/blob/master/notifications_catalog/templates/app-created.yaml shows an easy way to reference `message` in the subject. Does it work here?

Reply:
Yeah looks like that should work, done